internal/db/nosql/cassandra/option_test.go

```diff
@@ -19,1026 +19,703 @@ package cassandra
 
 import (
 	"crypto/tls"
+	"reflect"
 	"testing"
+	"time"
 
 	"github.com/gocql/gocql"
+	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/test/comparator"
 
 	"go.uber.org/goleak"
 )
 
+var (
+	// Goroutine leak is detected by `fastime`, but it should be ignored in the test because it is an external package.
+	goleakIgnoreOptions = []goleak.Option{
+		goleak.IgnoreTopFunction("github.com/kpango/fastime.(*Fastime).StartTimerD.func1"),
+	}
+
+	// default comparator option for client
+	clientOpts = []comparator.Option{
```

Comment:
I think `clientOpts` is general naming.
If we want to add code to the implementation file (_.go), I think naming may same idea.
so you should change name.
for example
```
clientOpts -> comparatorClientOpts
```